main.py

```diff
@@ -0,0 +1,32 @@
+import discord
+import os
+from discord.ext import commands
+import discord_components
+import ErrorHandling
+import Reactions
+import Help
+
+
+TOKEN = ''
+GUILD = 'GUILD'
+
+intents = discord.Intents.all()
```

Comment:
Why does the bot need access to _**all**_ intents? Bots should only have access to the specific intents they need. Giving them all intents is dangerous in the event the bot gets hacked or the token is stolen.

Reply:
fixed